movieland_persistence_jdbc/src/main/java/com/solovey/movieland/dao/jdbc/config/QueryConfig.java

```diff
@@ -123,30 +123,72 @@ public String updateMovieSql(){
     }
 
     @Bean
-    public String deleteGenreMappingsByIdsSql(){
+    public String deleteGenreMappingsByIdsSql() {
         return "delete from movies.genre_mapping " +
                 "where movie_id=:movieId " +
                 "and genre_id not in (:genreIds)";
     }
 
     @Bean
-    public String deleteCountryMappingsByIdsSql(){
+    public String deleteCountryMappingsByIdsSql() {
         return "delete from movies.country_mapping " +
                 "where movie_id=:movieId " +
                 "and country_id not in (:countryIds)";
     }
 
     @Bean
-    public String inserMovieRateSql(){
+    public String inserMovieRateSql() {
         return "insert ignore into movies.movie_rates(movie_id,user_id,rate) " +
                 "values (?,?,?)";
     }
 
     @Bean
-    public String getMoviesRatingsSql(){
+    public String getMoviesRatingsSql() {
         return "select movie_id,round(sum(rate)/count(*) ,1) rating,count(*) cnt " +
                 "from  movies.movie_rates mr " +
                 "group by movie_id";
     }
 
-}
+    @Bean
+    public String getMoviesForReportSql() {
+        return "select m.movie_id,movie_name_native as title,description," +
+                " GROUP_CONCAT(DISTINCT g.genre_name ORDER BY g.genre_id ASC SEPARATOR ',') AS genre," +
+                " price,added_date,modified_date,rating," +
+                " count(distinct r.user_id) as reviews_count" +
+                " from movies.movie m" +
+                " join movies.genre_mapping gm on m.movie_id=gm.movie_id " +
+                " join movies.genre g on gm.genre_id=g.genre_id " +
+                " left join movies.review r on m.movie_id=r.movie_id ";
+    }
```

Comment:
I didn't find SQL statement where you are updating MOVIE.RATING attribute value when saving movie_rates.
Is it contains correct value? 
If not, please consider movie_rates table in the query to calculate proper rating